src/reconstruction/enzyme_constrained.jl

```diff
@@ -1,20 +1,20 @@
 """
 $(TYPEDSIGNATURES)
 
-Wrap a model into a [`EnzymeConstrainedModel`](@ref), following the structure
-given by GECKO algorithm (see [`EnzymeConstrainedModel`](@ref) documentation for
-details). Multiple capacity constraints can be placed on the model using the
+Wrap a model into an [`EnzymeConstrainedModel`](@ref), following the structure
+given by the GECKO algorithm (see [`EnzymeConstrainedModel`](@ref) documentation
+for details). Multiple capacity constraints can be placed on the model using the
 kwargs.
```

Comment:
`kwargs` is pythonism and shortcut, just say "Optionally, gene products may be organized in groups with independent mass bounds."